### PR TITLE
Minimax grids with 26 to 32 grid points for low-scaling GW

### PR DESCRIPTION
**Pull request by JWilhelm**
Add minimax grids with 26, 28, 30, 32, 34 grid points that enable accuracy of < 0.01 eV for the GW100 test set. Remove mixed Clenshaw-Curtis/minimax grids that is less accurate and has higher computational cost.

### CODE REVIEW

File: src/minimax/minimax_rpa.F [hunk 1]
```diff
@@ -10111,4 +10111,6526 @@ SUBROUTINE get_fit_coef_k20(Rc, L_b, U_b, fit_coef)
...
    END SUBROUTINE get_fit_coef_k20
 
+! **************************************************************************************************
+!> \brief ...
+!> \param k ...
+!> \param E_range ...
+!> \param aw ...
+! **************************************************************************************************
+   SUBROUTINE get_rpa_minimax_coeff_larger_grid(k, E_range, aw)
+      INTEGER                                            :: k
```
Comment (fstein93) [inline src/minimax/minimax_rpa.F:10121]: Please add intents.

File: src/minimax/minimax_rpa.F [hunk 2]
```diff
@@ -10111,4 +10111,6526 @@ SUBROUTINE get_fit_coef_k20(Rc, L_b, U_b, fit_coef)
...
+      REAL(KIND=dp)                                      :: E_range
+      REAL(KIND=dp), ALLOCATABLE, DIMENSION(:)           :: aw
```
Comment (fstein93) [inline src/minimax/minimax_rpa.F:10123]: Does aw have to be allocatable?

File: src/minimax/minimax_rpa.F [hunk 3]
```diff
@@ -10111,4 +10111,6526 @@ SUBROUTINE get_fit_coef_k20(Rc, L_b, U_b, fit_coef)
...
+      CALL timestop(handle)
+
+   END SUBROUTINE get_rpa_minimax_coeff_larger_grid
+
+! **************************************************************************************************
+!> \brief ...
+!> \param aw ...
+!> \param E_ratio ...
+! **************************************************************************************************
+   SUBROUTINE rescale_grid(aw, E_ratio)
```
Comment (fstein93) [inline src/minimax/minimax_rpa.F:10159]: You can declare this routine PURE. (Same for get_coeff_* routines)

File: src/minimax/minimax_exp_gw.F [hunk 4]
```diff
@@ -0,0 +1,7144 @@
...
+      CALL timestop(handle)
+
+   END SUBROUTINE get_exp_minimax_coeff_gw
+
+! **************************************************************************************************
+!> \brief ...
+!> \param aw ...
+!> \param E_ratio ...
+! **************************************************************************************************
+   SUBROUTINE rescale_grid(aw, E_ratio)
```
Comment (fstein93) [inline src/minimax/minimax_exp_gw.F]: This routine can be declared PURE (same with get_coeff_* routines).

File: src/minimax/minimax_rpa.F (hunk 1 above)
Comment (JWilhelm) [reply src/minimax/minimax_rpa.F:10121]: Can you please explain what exactly you are meaning? All your other points below are clear to me and will be addressed.

Comment (fstein93) [reply src/minimax/minimax_rpa.F:10121]: You have added intents in minimax_exp_gw.F, but not here. (INTENT(IN), INTENT(OUT), INTENT(INOUT))

Comment (JWilhelm) [reply src/minimax/minimax_rpa.F:10121]: oh, I see, I will add, thanks!

**Pull request merged**